### PR TITLE
[NP4423] Replace  Notification Messages 

### PR DESCRIPTION
**Pull request by tala-abuadas**

### CODE REVIEW

File: src/foam/nanos/auth/RetrievePassword.js [hunk 1]
```diff
@@ -93,7 +94,8 @@ foam.CLASS({
         const user = this.User.create({ email: this.email });
         this.resetPasswordToken.generateToken(null, user).then((_) => {
           this.ctrl.add(this.NotificationMessage.create({
-            message: `${this.INSTRUC_ONE} ${this.email}. ${this.INSTRUC_TWO}`,
+            message: `${this.INSTRUC_ONE_TITLE}`,
```
Comment (jlhughes) [inline src/foam/nanos/auth/RetrievePassword.js:97]: I think this is ridiculous in terms of redundancy, but if you @sarthak-marwaha think this is appropriate UI/UX, then fine.

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/approval/ApprovalRequest.js [hunk 2]
```diff
@@ -623,8 +623,8 @@
       message: 'You have successfully assigned this request'
     },
      {
-       name: 'SUCCESS_ASSIGNED_TITLE',
-       message: 'Request Assigned'
+      name: 'SUCCESS_ASSIGNED_TITLE',
```
Comment (sarthak-marwaha) [inline src/foam/nanos/approval/ApprovalRequest.js:626]: Does this already have a translation?

Comment (tala-abuadas) [reply src/foam/nanos/approval/ApprovalRequest.js:626]: yes

File: src/foam/nanos/auth/RetrievePassword.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ foam.CLASS({
   ],
 
   messages: [
+    { name: 'INSTRUC_ONE_TITLE', message: 'Password Reset Instructions Sent' },
     { name: 'INSTRUC_ONE', message: 'Password reset instructions were sent to' },
```
Comment (sarthak-marwaha) [inline src/foam/nanos/auth/RetrievePassword.js:27]: Do we still use this anywhere else?

File: src/foam/nanos/auth/RetrievePassword.js (hunk 1 above)
Comment (sarthak-marwaha) [reply src/foam/nanos/auth/RetrievePassword.js:97]: It is still two messages just split up into two rows now. I believe we can delete the 'INSTRUC_ONE' if it not required

File: src/foam/nanos/auth/RetrievePassword.js (hunk 3 above)
Comment (tala-abuadas) [reply src/foam/nanos/auth/RetrievePassword.js]: no

**Review by jlhughes — APPROVED** [review]

**Pull request merged**